docs/docs/overview/why_secretless.md

```diff
@@ -6,32 +6,36 @@ description: Secretless Broker Documentation
 
 It can be a losing battle to try and train every user and application developer about how to safely handle a secret once they’ve obtained it even if you can mitigate all of the attack surfaces of the client app itself. There are so many things that can go wrong, and people (and applications) have better things to worry about than babysitting secrets (such as doing their jobs). Do you want developers to write features or try to be security engineers too?
 
-The Secretless Broker is specifically written to be really good at interacting with a variety of vaults and abstracting away the security to a specialized module that can easily be audited and expanded with custom plugins. It does a really good job of securing any secrets that it might obtain from a vault, and it does it the same way no matter the developer, technology, or platform. Additionally, it knows what to do when a secret has been changed so that the client users and code never have to respond to changing secrets or even know that they are changing.
+The Secretless Broker is specifically written to be really good at interacting with a variety of vaults and abstracting away the security to a specialized module that can easily be audited and expanded with custom plugins. It does a really good job of securing any secrets that it might obtain from a vault, and it does it the same way no matter the developer, technology, or platform.
 
 In summary, when the client connects to the target service through the Secretless Broker:
 
+- **The client does not have to know how to fetch credentials**
+
+  The client/app no longer has to directly interact with a secrets vault
+  or know how to code its APIs. No code must be changed to fetch credentials before
+  opening a connection. If the vault used changes when the environment or platform
+  changes no changes in code are required, thus reducing the probability of human
+  error and bugs.
+
 - **The client is not part of the threat surface**
 
   The client/app no longer has direct access to the password, and therefore cannot reveal it.
 
 - **The client doesn't have to know how to properly manage secrets**
 
   Handling secrets safely involves some complexities, and when every application needs to know how to handle secrets, accidents happen. The Secretless Broker centralizes the client-side management of secrets into one code base, making it easier for developers to focus on delivering features.
-
-- **The client doesn't have to manage secret rotation**
-
-  The Secretless Broker is responsible for establishing connections to the backend, and can handle secrets rotation in a way that's transparent to the client.
```

Comment:
Hmm.. why wouldn't we want to highlight the seamless rotation?

Reply:
it's not all that seamless yet. secretless doesn't pick up when credentials are changed and there is still the possibility of a race condition where it retrieves the cred and then it's changed as it's opening the connection. there is work to be done yet to make this work nicely, and we will do it, and then we can shout it from the rooftops.